src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2288,7 +2288,12 @@ private string GetCurrentUserName()
 
                 // At this point, we should have a suspended process.  Get the .Net Process object, resume the process, and return.
                 Process result = Process.GetProcessById(lpProcessInformation.dwProcessId);
-                PlatformInvokes.ResumeThread(lpProcessInformation.hThread);
+                uint returnValue = PlatformInvokes.ResumeThread(lpProcessInformation.hThread);
+
+                if (returnValue == 4294967295) // same as (DWORD)-1
```

Comment:
We should not use magic numbers in the code. Please define a descriptive constant for this value.